components/builder-api/src/server/mod.rs

```diff
@@ -76,33 +73,27 @@ features! {
 
 // Application state
 pub struct AppState {
-    config:         Config,
-    packages:       S3Handler,
-    github:         GitHubClient,
-    jobsrv:         RpcClient,
-    oauth:          OAuth2Client,
-    memcache:       RefCell<MemcacheClient>,
-    artifactory:    ArtifactoryClient,
-    db:             DbPool,
-    event_producer: Option<Box<dyn EventPublisher>>,
+    config:      Config,
+    packages:    S3Handler,
+    github:      GitHubClient,
+    jobsrv:      RpcClient,
+    oauth:       OAuth2Client,
+    memcache:    RefCell<MemcacheClient>,
+    artifactory: ArtifactoryClient,
+    db:          DbPool,
 }
 
 impl AppState {
     pub fn new(config: &Config, db: DbPool) -> error::Result<AppState> {
-        let mut app_state =
+        let app_state =
             AppState { config: config.clone(),
                        packages: S3Handler::new(config.s3.clone()),
                        github: GitHubClient::new(config.github.clone())?,
                        jobsrv: RpcClient::new(&format!("{}", config.jobsrv)),
                        oauth: OAuth2Client::new(config.oauth.clone())?,
                        memcache: RefCell::new(MemcacheClient::new(&config.memcache.clone())),
                        artifactory: ArtifactoryClient::new(config.artifactory.clone())?,
-                       db,
-                       event_producer: None };
-
-        if feat::is_enabled(feat::EventBus) {
```

Comment:
Since we are doing away with this feature, we should also delete lines 108 and 117 here.

Reply:
Agree - will update.